web/src/testgrid-grid-header-row.ts

```diff
@@ -0,0 +1,34 @@
+import { LitElement, html, css } from "lit";
```

Comment:
IMO `testgrid-grid-header-row` and `testgrid-grid-row-header` are a little bit confusing. perhaps changing the naming would help with that